src/components/CodeEditor/CodeEditor.js

```diff
@@ -59,7 +59,10 @@ class CodeEditor extends Component {
           O Babel não pôde ser carregado.
           <br />
           <br />
-          Isto pode ser causado por algum bloqueador de anúncio. Se você está usando algum, considere adicionar o endereço reactjs.org na <i>whitelist</i>, ou lista de permitidos, e feito isso os exemplos com <i>live code</i> irão funcionar.
+          Isto pode ser causado por algum bloqueador de anúncio. Se você está
```

Comment:
please keep this text all on the same line

Reply:
I didn't change this file! I just worked on the refs-and-the-dom.md file, and just executed the `yarn check-all`command, that maybe did the change on this file.